rpmlint/__init__.py

```diff
@@ -87,7 +87,9 @@ def process_lint_args(argv):
     parser.add_argument('-V', '--verbose', action='store_true', help='provide detailed explanations where available')
     parser.add_argument('-p', '--print-config', action='store_true', help='print the settings that are in effect when using the rpmlint')
     parser.add_argument('-i', '--installed', nargs='+', default='', help='installed packages to be validated by rpmlint')
-    parser.add_argument('-s', '--strict', action='store_true', help='treat all messages as errors')
+    lint_modes_parser = parser.add_mutually_exclusive_group()
+    lint_modes_parser.add_argument('-s', '--strict', action='store_true', help='treat all messages as errors')
+    lint_modes_parser.add_argument('-P', '--permissive', action='store_true', help='error only if badness threshold is exceeded')
```

Comment:
Not sure if just not say 'treat all errors as non-fatal and do not fail' or something along the way.

Reply:
Sure, makes sense.